pages/reference-modules/4337/v0.3.0-1/operations/validateUserOp.mdx

```diff
@@ -0,0 +1,111 @@
+import { Tabs, Callout } from 'nextra/components'
+
+import LegacyCallout from '../../../../../components/callouts/LegacyCallout.mdx'
+import OnlySafeTxCallout from '../../../../../components/callouts/OnlySafeTxCallout.mdx'
+import ReentrancyCallout from '../../../../../components/callouts/ReentrancyCallout.mdx'
+import IrreversibilityCallout from '../../../../../components/callouts/IrreversibilityCallout.mdx'
+
+
+
+# `validateUserOp` (0x19822f7c)
+
+
+
+Validates a user operation provided by the entry point.
```

Comment:
One issue with this documentation is that it is inherited from the `IAccount` interface from the official ERC-4337 repo. It is more meant for implementors (so our team) and not for users of the interface itself (so the target of this documentation).

Not an issue for this PR, but I would argue that this is something to consider for the automated doc generation - that for some functions you may want to manually specify the documentation instead of reading it from NatSpec.